src/image-tags.js

```diff
@@ -54,6 +54,34 @@ function pickImage(pageType, data, url) {
   }
 }
 
+function pickAttributionFromCard (story, cardId) {
+  const { metadata = {} } = story.cards.find(card => card.id === cardId) || {};
+  if(metadata && !isEmpty(metadata) && get(metadata, ['social-share', 'image', 'attribution'], false)) {
+    return metadata['social-share'].image.attribution || undefined;
+  }
+}
+
+function pickAttributionFromStory(story){
+  return story["hero-image-attribution"] || story.summary || get(story, ["alternative", "home", "default", "headline"]) || story.headline;
```

Comment:
Taking story `hero-image-attribution` at the first place if it's not present then taking summary(social share message) if this also not present then taking alternate headline, and default will be story headline.

Reply:
same here as well